### DISCUSSION TITLE
Bitmap font loader for mutiple font atlas

### DISCUSSION THREAD

**Discussion by DongkunLee** [raysan5/raylib#3503] [General]
I made this helper 

[Raylib bitmap font extender](https://github.com/DongkunLee/raylibbmfontextender )

I think this will help for  unicode loading.

**Comment by raysan5** [comment]
@DongkunLee Nice improvement! Actually that function was reviewed yesterday to avoid some security concerns. 

If you want, feel free to integrate your changes into raylib and send a PR.

**Reply by DongkunLee** [reply]
I can't understand 

`  If you want, feel free to integrate your changes into raylib and send a PR.  `
meaning. 

Anyway. I really want my changes integrated to raylib!!  

at this point, i don't know how to contribute. 

I will learn contributing to raylib.  by github.

**Reply by DongkunLee** [reply]
I got what PR means. 

Do you mean Pull Request? 

I will do it.

**Reply by DongkunLee** [reply]
I will pull request later. 

Don't know how to use it

**Reply by raysan5** [reply]
@DongkunLee No worries, take your time, the process works as follow:

 - Fork raylib into your user account
 - Modify your raylib fork integrating your changes
 - Test your changes to make sure everything works
 - Review code to follow raylib [coding conventions](https://github.com/raysan5/raylib/blob/master/CONVENTIONS.md)
 - Send a Pull Request from your fork into main raylib repository
 - Wait for me to review the PR, I could ask you for some changes
 - In case of changes, you can just apply to your branch and update the PR
 - If everything is right, changes will be merged into raylib

**Reply by ghost** [reply]
did this ever make it in
